pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -88,6 +88,7 @@ def __init__(
         allgather_bucket_size: int = 2e8,
         reduce_bucket_size: int = 2e8,
         zero_allow_untested_optimizer: bool = True,
+        logging_batch_size_per_gpu: Optional[int] = 1,
```

Comment:
```suggestion
        logging_batch_size_per_gpu: int = 1,
```

Reply:
If set to None, the logging_batch_size_per_gpu is inferred from the train DataLoader's BatchSampler

Reply:
How about we follow the paradigm of using "auto" as infer?
```suggestion
        logging_batch_size_per_gpu: Union[int, str] = 1
```

Reply:
Thanks @SeanNaren. Using `"auto"` sounds good to me and will result in easier to understand config files.